src/main/java/controllers/components/QuestionsMngr.java

```diff
@@ -30,7 +31,13 @@
 //
 //    }
 
```

Comment:
This might be helpful (if we are refactoring  the game.) 
https://www.oodesign.com/strategy-pattern.html

Reply:
Yes, totally agree! When the prof was talking about "bad smells" in lecture this week I thought back to the "if" statement in the constructor and how that is a prime candidate for refactoring. :) :) :)